ucum-java8/src/main/java/systems/uom/ucum/UCUM.java

```diff
@@ -768,9 +786,116 @@ public static UCUM getInstance() {
 	/** As per <a href="http://unitsofmeasure.org/">UCUM</a> standard. */
 	public static final Unit<Power> HORSEPOWER = addUnit(new ProductUnit<Power>(
 			FOOT_INTERNATIONAL.multiply(POUND_FORCE).divide(SECOND)));
-	// ///////////////////////////////////////////////////////
-	// SECTIONS §44-§46 skipped; implement later if needed //
-	// ///////////////////////////////////////////////////////
+	
+	////////////////////////////////////////////
```

Comment:
hahaha, yeeah, you're right.. I'm trying to not use any formatter for this project as everything would be a mess on diff.